test/pkg/environment/azure/default_aksnodeclass.yaml

```diff
@@ -3,4 +3,4 @@ kind: AKSNodeClass
 metadata:
   name: default
 spec:
-  imageFamily: Ubuntu2204
+  imageFamily: Ubuntu
```

Comment:
Is this a holdover from one of your other PRs?

Maybe split into a separate smaller PR?
Feel free to ignore if that's annoying to do.